### PR TITLE
debian-base: purge libsystemd0 to eliminate CVE false-positives

### PR DESCRIPTION
**Pull request by ixdy**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**: we already purge `systemd` and `systemd-sysv` from the `debian-base` container, since we don't use them, but `libsystemd0` is still part of the image, which is causing false CVE alerts from the GCR Container Analysis service.

**Does this PR introduce a user-facing change?**:

```release-note
Images based on debian-base no longer include the libsystemd0 package. This should have no user-facing impact.
Additionally, the addon-manager image is updated to use kubectl v1.11.3.
```

I don't expect this change to have any user-facing changes. We'll need to be more cautious once we update debian-iptables and debian-hyperkube-base; there shouldn't be any issues, but hyperkube in particular has surprised me before.

/assign @ihmccreery @awly @tallclair

### CODE REVIEW

**Comment by awly** [comment]
/lgtm
Thanks for cleaning this up!

**Comment by ixdy** [comment]
/hold

I forgot to add the sig, but I've already promoted the debian-base image, so I'm going to bump the child images in this PR too.

File: cluster/addons/addon-manager/Makefile [hunk 1]
```diff
@@ -15,10 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.8
-KUBECTL_VERSION?=v1.10.7
+VERSION=v8.9
+KUBECTL_VERSION?=v1.10.9
```
Comment (ixdy) [inline cluster/addons/addon-manager/Makefile]: @MrHohn does this version of kubectl still make sense? or should we bump to 1.11.x?

**Comment by ixdy** [comment]
pushed a commit bumping the debian-base references, PTAL.

File: cluster/addons/addon-manager/Makefile (hunk 1 above)
Comment (MrHohn) [reply cluster/addons/addon-manager/Makefile]: Thanks for adding me, bumping to 1.11.x sounds good to me.

Comment (ixdy) [reply cluster/addons/addon-manager/Makefile]: updated to kubectl v1.11.3.

File: build/debian-iptables/Makefile [hunk 2]
```diff
@@ -16,12 +16,12 @@
 
 REGISTRY?="staging-k8s.gcr.io"
 IMAGE=$(REGISTRY)/debian-iptables
-TAG?=v10.2
+TAG?=v11.0
```
Comment (awly) [inline build/debian-iptables/Makefile]: why not `v10.3` ?

Comment (ixdy) [reply build/debian-iptables/Makefile]: I generally reserve the .N bumps for rebases of the existing image. We actually removed a package here.

**Comment by ixdy** [comment]
/assign @zmerlynn 

for cluster/ approval

**Comment by Random-Liu** [comment]
This actually breaks node-problem-detector. :P https://github.com/kubernetes/node-problem-detector/issues/232

Is there any other way to fix the CVE?

**Comment by ixdy** [comment]
I'd rather not add libsystemd back to the base image, since we don't need it in the core kubernetes images.

If you need it in your image, you should be able to add it back with `clean-install`.

**Pull request merged**